EZSwiftExtensionsTests/DateTests.swift

```diff
@@ -229,67 +241,55 @@ class DateTests: XCTestCase {
     }
     
     func testEra() {
-        let customDate = Date(fromString: "12-01-2015 05:45:12", format: self.format)
-        XCTAssertEqual(customDate?.era, Calendar.current.component(Calendar.Component.era, from:customDate!))
+        XCTAssertEqual(date.era, Calendar.current.component(Calendar.Component.era, from: date))
     }
     
     func testYear() {
-        let customDate = Date(fromString: "12-01-2015 05:45:12", format: self.format)
-        XCTAssertEqual(customDate?.year, 2015)
+        XCTAssertEqual(self.date.year, 2015)
     }
     
     func testMonth() {
-        let customDate = Date(fromString: "09-01-2015 05:45:12", format: self.format)
-        XCTAssertEqual(customDate?.month, 1)
+        XCTAssertEqual(self.date.month, 11)
     }
     
     func testMonthAsString() {
-        let customDate = Date(fromString: "09-01-2015 05:45:12", format: self.format)
-        XCTAssertEqual(customDate?.monthAsString, "January")
-        
-        let customDate2 = Date(fromString: "10-03-2009 05:45:12", format: self.format)
-        XCTAssertEqual(customDate2?.monthAsString, "March")
-        
-        let customDate3 = Date(fromString: "11-11-2012 05:45:12", format: self.format)
-        XCTAssertEqual(customDate3?.monthAsString, "November")
+        XCTAssertEqual(self.date.monthAsString, "November")
     }
     
     func testWeekDay() {
-        let customDate = Date(fromString: "09-01-2015 05:45:12", format: self.format)
-        XCTAssertEqual(customDate?.weekday, "Friday")
+        XCTAssertEqual(self.date.weekday, "Sunday")
+        NSTimeZone.default = TimeZone(abbreviation: "UTC")!
```

Comment:
Good call. I think when I wrote those tests I only cared about the build server. This looks better.